Fixed activity indicator that was always visible and not animating

lokeshub opened the pull request:
On Add Location screen, now activity indicator is stopped and hidden when query not running and animates when querying.

Commit:
Fixed activity indicator that was always visible and not animating on Add Location screen

alokc83 commented on WeatherInfo/ViewControllers/Locations View Controller/Add Location View Controller/AddLocationViewController.swift:
```diff
@@ -38,19 +38,21 @@ class AddLocationViewController: UIViewController {
             self.tableView.reloadData()
         }
-        
+        self.activityIndicatorView.isHidden = true
```
Why this is needed when you are changing states in ViewDidAppear?

lokeshub replied:
When I ran in slow animation it is visible there unanimated when screen is loaded and disappears when the first time search is made.

lokeshub committed:
Corrected one spelling
On main story board activity indicator's property, selected "Hide when Stopped'. No need to manually hide after this.

lokeshub commented on WeatherInfo/ViewControllers/Locations View Controller/Add Location View Controller/AddLocationViewController.swift:
```diff
@@ -38,19 +38,21 @@ class AddLocationViewController: UIViewController {
             self.tableView.reloadData()
         }
-        
+        self.activityIndicatorView.isHidden = true
```
On main story board activity indicator's property, selected "Hide when Stopped'. No need to manually hide in code after this.

alokc83 replied:
What about the bug that you mentioned yesterday?

lokeshub replied:
This was introducing that bug. The activity indicator was already not hidden when the view loaded. Once we set this above property, we have to just start/Stop the animation to make the activity indicator view they way it should work.

alokc83 replied:
Then hidden is not the way. you should or should not be animating. 
As property says hide when stop animating. 
@lokeshub

alokc83 commented on WeatherInfo/ViewControllers/Locations View Controller/Add Location View Controller/AddLocationViewController.swift:
```diff
@@ -38,19 +38,21 @@ class AddLocationViewController: UIViewController {
         }
-    }
-
-    override func viewDidAppear(_ animated: Bool) {
```
Why did you remove viewDidAppear

alokc83 replied:
I would want searchBar to become the first responder when the view is being presented.

lokeshub replied:
Yes it is searchBar is made first responder still in viewDidAppear. I did not remove it.

alokc83 commented:
What have you done to fix the issue?

lokeshub commented:
ActivityIndicator's property in storyboard to hide when not animating was enabled.
And this status change was not required in code after that

alokc83 commented:
Read my comment @lokeshub

lokeshub commented:
Closing this as invalid changes. I will work on fresh PR.
